test/built-ins/Error/message_property.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2015 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Error constructor creates own message property
+info: >
+  19.5.1.1 Error ( message )
+
+  ...
+  4.
+    ...
+    c. Let msgDesc be the PropertyDescriptor{[[Value]]: msg, [[Writable]]: true, [[Enumerable]]: false, [[Configurable]]: true}.
+    d. Let status be DefinePropertyOrThrow(O, "message", msgDesc).
+es6id: 19.5.1.1
+---*/
+
+var message = "my-message";
+var error = new Error(message);
+var desc = Object.getOwnPropertyDescriptor(error, "message");
+
+assert.sameValue(desc.value, message, "Property descriptor [[Value]]");
+assert.sameValue(desc.writable, true, "Property descriptor [[Writable]]");
+assert.sameValue(desc.enumerable, false, "Property descriptor [[Enumerable]]");
+assert.sameValue(desc.configurable, true, "Property descriptor [[Configurable]]");
```

Comment:
These should use the `verify*` property helper functions: 

```
includes:
    - propertyHelper.js
```

```js
verifyWritable(error, "message");
verifyNotEnumerable(error, "message");
verifyConfigurable(error, "message");
```
Example: https://github.com/tc39/test262/blob/master/test/built-ins/Array/from/Array.from-descriptor.js